icu4c/source/test/intltest/itrbnf.cpp

```diff
@@ -2605,6 +2606,37 @@ IntlTestRBNF::TestNumberingSystem() {
     }
 }
 
+void
+IntlTestRBNF::TestInfiniteRecursion() {
+    UnicodeString badRules[] = {
+        ">>",
```

Comment:
could we also add the case of ">>>" and "<<<". thanks

Reply:
Done!